pydocstringformatter/_configuration/toml_parsing.py

```diff
@@ -45,11 +45,18 @@ def parse_toml_option(
         if value is True:
             return [action.option_strings[0]]
         return []
+
     if isinstance(action, argparse._StoreAction):
         if isinstance(value, int):
             value = str(value)
         return [action.option_strings[0], value]
-    return []  # pragma: no cover
+
+    if isinstance(action, argparse._ExtendAction):  # type: ignore[attr-defined]
```

Comment:
I added the `argparse.` notation as we also have some actions that don't come from `argparse`. This makes this a bit clearer.